package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "react-accordion",
+  "version": "1.0.0",
```

Comment:
If we are going to release this on NPM, we shouldn't make this 1.0.0.
Maybe 0.1.0?